x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/utils.test.ts

```diff
@@ -5,14 +5,29 @@
  */
 
 import moment from 'moment';
+import sinon from 'sinon';
 
-import { generateId, parseInterval, getDriftTolerance, getGapBetweenRuns } from './utils';
+import {
+  generateId,
+  parseInterval,
+  parseScheduleDates,
+  getDriftTolerance,
+  getGapBetweenRuns,
+} from './utils';
 
 describe('utils', () => {
+  const anchor = '2020-01-01T06:06:06.666Z';
+  const unix = moment(anchor).valueOf();
   let nowDate = moment('2020-01-01T00:00:00.000Z');
+  let clock: sinon.SinonFakeTimers;
 
   beforeEach(() => {
     nowDate = moment('2020-01-01T00:00:00.000Z');
+    clock = sinon.useFakeTimers(unix);
+  });
+
+  afterEach(function() {
```

Comment:
nit: I would do a regular () like `afterEach(() => {` here and avoid function.

Reply:
Updating!